dev/automation-hub/initial_data.json

```diff
@@ -4,7 +4,7 @@
     "pk": 1,
     "fields": {
       "name": "system:partner-engineers",
-      "permissions": []
+      "permissions": [37, 266, 267, 270, 287, 288, 289, 290, 283, 284, 285, 286]
```

Comment:
Where do these magic numbers come from?

Reply:
PKs for permissions. These are the specific permissions I used

https://github.com/ansible/galaxy_ng/pull/348/files#diff-a052008809a09a99fd60d724a42a3c66R54

I don't know if they are all the ones we actually want partner engineers to have, but those are the ones that were needed for the tests to pass.

Reply:
I just tried to leave the group names as comments, but JSON doesn't allow comments